lib/search/index.ts

```diff
@@ -364,6 +364,11 @@ export const middleware: S.Middleware = (store) => {
       }
 
       case 'CREATE_NOTE_WITH_ID':
+        searchState.showTrash = false;
+        searchState.notes.set(action.noteId, toSearchNote(action.note ?? {}));
+        indexNote(action.noteId);
+        queueSearch();
+        return next(action);
```

Comment:
I'm not confident this is the best way or best place to do this. It seems to work but would love thoughts on if there is a better way. @dmsnell

Reply:
yeah this is something we have to be careful about since we're changing the search property for trash. we would only want to do this if we know that the `showTrash` reducer is also turning off the trash view when creating a note

Reply:
Thanks @dmsnell If I understand things properly it does look like we are doing that [here](https://github.com/Automattic/simplenote-electron/blob/update/disallow-creating-note-from-trash/lib/state/ui/reducer.ts#L305). Do I have that right?

Reply:
yep that's right 👍